Makefile.work

```diff
@@ -153,6 +162,31 @@ ifneq ($(SONIC_DPKG_CACHE_SOURCE),)
 	DOCKER_RUN += -v "$(SONIC_DPKG_CACHE_SOURCE):/dpkg_cache:rw"
 endif
 
+
+# User name and tag for "docker-*" images created by native dockerd mode.
+ifeq ($(strip $(SONIC_CONFIG_USE_NATIVE_DOCKERD_FOR_BUILD)),y)
+DOCKER_USERNAME = -$(shell echo $(USER) | tr '[:upper:]' '[:lower:]')
+DOCKER_USERTAG = $(SLAVE_TAG)
```

Comment:
why use SLAVE_TAG here? why not use latest? are you assuming same user have multiple build?

Reply:
The SLAVE_TAG hash is generated uniquely for Particular User, View, Configured platform and Branch name.  This solves the parallel build of the same user from a different view.